src/api/files.js

```diff
@@ -1,9 +1,17 @@
-import {handleResponse} from './base';
+import {handleResponse, defaultUrl} from './base';
 
 export const FilesApi = {
     uploadFile(jobId, file) {
         const body = new FormData();
         body.append('file', file);
-        return fetch(`/api/jobs/${jobId}/files`, { method: 'POST', body }).then(handleResponse);
+        return fetch(`${defaultUrl}/api/jobs/${jobId}/files`, { method: 'POST', body }).then(handleResponse);
+    },
+    uploadLocalFile(jobId, filePath) {
+        const body = JSON.stringify({type: 'LOCAL', path: filePath});
+        return fetch(`${defaultUrl}/api/jobs/${jobId}/files`, {
+            headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+          }, method: 'POST', body }).then(handleResponse);
```

Comment:
Would be nice to reformat this a bit:

```suggestion
          return fetch(`${defaultUrl}/api/jobs/${jobId}/files`, {
              method: 'POST',
              headers: {
                  'Accept': 'application/json',
                  'Content-Type': 'application/json'
              },                
              body 
          }).then(handleResponse);
```